app/screens/snack_bar/index.tsx

```diff
@@ -85,6 +95,9 @@ const SnackBar = ({
     componentId,
     onAction,
     sourceScreen,
+    autoClose = true,
```

Comment:
I am not a big fan of defaults set to true (in my head model, omitting something should mean you don't want it). 0/5 whether we should rename the variable so it means the opposite and can be defaulted to false. For example, `keepOpen` (but open to other options, English is hard).

Reply:
I'm just preserving the default behavior, which is the snackbar closes automatically. I've now renamed the param to `keepOpen` so the default is false, as desired, and existing behavior is maintained.